seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcStarRocksdbIT.java

```diff
@@ -111,7 +115,12 @@ JdbcCase getJdbcCase() {
     }
 
     @Override
-    void compareResult(String executeKey) {}
+    void checkResult(String executeKey, TestContainer container, Container.ExecResult execResult) {
+        if (container.identifier().equals(TestContainerId.SEATUNNEL)) {
+            Assertions.assertTrue(
+                    execResult.getStdout().contains("Loading catalog tables for catalog"));
+        }
```

Comment:
i run some test, find the stdout and container log is different on different engine and version.
so here only check seatunnel engine's log.

Reply:
when has this log `Loading catalog tables for catalog` means it was load table from MySQLCatalog. rather than 
use jdbc. 

related log location: 
https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/utils/JdbcCatalogUtils.java#L77


https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/utils/JdbcCatalogUtils.java#L127